### DISCUSSION TITLE
CVE-2020-15250 - Detected in jakarta.el (3.0.4) - Javax to Jakarta migration in Dropwizard Validation

### DISCUSSION THREAD

**Discussion by kvishrutha** [dropwizard/dropwizard#6107] [Q&A] [answered]
There are some CVE's in Jakarta.el (3.0.4) - https://mvnrepository.com/artifact/org.glassfish/jakarta.el/3.0.4 
Currently dropwizard-validation is using Javax - https://github.com/dropwizard/dropwizard/blob/release/2.1.x/dropwizard-validation/src/main/java/io/dropwizard/validation/BaseValidator.java#L7

I have seen some closed PR's to bump Jakarta.el dependancy version - https://github.com/dropwizard/dropwizard/pull/3612 , https://github.com/dropwizard/dropwizard/pull/3505 
But they are blocked and closed.

Wondering, when will the migration happen?

**Comment by joschi** [comment, marked as answer]
CVE-2020-15250 is a false positive on Jakarta.EL.

**Comment by kvishrutha** [comment]
Hi @joschi  Thanks for the update.
Dropwizard is currently using Jakarta.el version 3.0.4.
The CVE is not detected in the future versions of jakarta.el.
When is dropwizard going to upgrade the dependency.

**Reply by joschi** [reply]
It is a false positive. It will not be detected once this has been resolved in whatever scanner you are using.

**Reply by kvishrutha** [reply]
Currently it is being detected in our scanner. I didn't understand the resolution you are talking about.

**Reply by kvishrutha** [reply]
@joschi  do you mean that Jakarta.el 3.0.4 has CVE resolved by itself?

**Reply by joschi** [reply]
No, Jakarta.EL has never been vulnerable to this CVE at all.

It is a false positive and mvnrepository.com is wrong about showing it for Jakarta.EL 3.0.4.

**Reply by kvishrutha** [reply]
@joschi  Thank you for the update :)
